src/main/scala/io/iohk/ethereum/blockchain/sync/fast/PivotBlockSelector.scala

```diff
@@ -35,16 +35,16 @@ class PivotBlockSelector(
   override def receive: Receive = idle
 
   def idle: Receive = handleCommonMessages orElse { case SelectPivotBlock =>
```

Comment:
WDYT about extracting method from code used in orElse?
I would go even further and extract methods for both branches in `if` inside.